Droidcon-Boston/app/src/main/java/com/mentalmachines/droidcon_boston/views/MainActivity.kt

```diff
@@ -31,6 +38,18 @@ class MainActivity : AppCompatActivity() {
         navView.setCheckedItem(R.id.nav_agenda)
     }
 
+    /*Callback from AgendaDetailFragment bookmark fab*/
+    override fun bookmarkStateChanged() {
+        var agendaFragmentTag : String = resources.getString(R.string.str_agenda)
+        val agendaFragment = supportFragmentManager.findFragmentByTag(agendaFragmentTag)
+
+        if (agendaFragment is AgendaFragment) {
+            val agendaDayFragment = agendaFragment.getCurrentregisteredFragment()
```

Comment:
small nit about naming...should be "getCurrentRegisteredFragment" because Java methods are usually camel cased